systems/common/msgbus/routing_key.go

```diff
@@ -145,3 +140,30 @@ func (r RoutingKeyBuilder) MustBuild() string {
 	}
 	return res
 }
+
+func Parse(s string) (RoutingKey, error) {
+
+	parts := strings.Split(s, ".")
+	if len(parts) != 5 {
+		return "", fmt.Errorf("invalid route %s", s)
```

Comment:
Please use logrus to output errors instead.

Reply:
returning error here.